### DISCUSSION TITLE
How do i reverse or cancel on deployment from my github account

### DISCUSSION THREAD

**Discussion by emmanuelofor** [vercel/community#2982] [Help]
### Summary

I did a deployment with vercel from my repository on github and I want to cancel it. How do I go about it?

### Steps to Reproduce

I did a deployment with vercel from my repository on github and I want to cancel it. How do I go about it?

**Comment by amyegan** [comment]
Hi @emmanuelofor. You can find a list of deployments for your project on your Dashboard. Deployments that have not finished can be cancelled from there. And older deployments can be redeployed to replace more recent accidental or problematic ones. You can learn more in our documentation about managing deployments: https://vercel.com/docs/concepts/deployments/managing-deployments
